resources/css/bem/simple-menu.less

```diff
@@ -137,6 +137,8 @@
 
   &__extra {
     background-color: hsl(var(--hsl-b5));
+    display: flex;
```

Comment:
actually this seems to just be unnecessary since the new component is `display: block` :thinking: 
i cant remember but this was probably leftover from testing a different layout.

Reply:
the click/tap target is different for mobile menu. it's also structurally more correct as the container isn't supposed to be part of the layout

Reply:
okay that makes sense, but don't I still need the flex from before for desktop then? or maybe I'm getting confused o-o 2479435b7db0497216093436ade5b4650f12ad9d

Reply:
hmm right, the buttons don't have `display: block` set on them (nor the correct width).